policy/daemon.go

```diff
@@ -141,45 +146,48 @@ func (p *policyd) Start(ctx context.Context) <-chan error {
 
 // Update updates and cache policy data
 func (p *policyd) Update(ctx context.Context) error {
-	glg.Info("Updating policy")
-	defer glg.Info("Updated policy")
+	jobID := fastime.Now().Unix()
```

Comment:
can we delete this jobID?

Reply:
It is easier to track the start and end from the log.
Better to used UUID other than timestamp, just that timestamp is simpler this time.